aws-android-sdk-iot/src/main/java/com/amazonaws/mobileconnectors/iot/AWSIotTLSSocketFactory.java

```diff
@@ -110,7 +109,7 @@ private Socket enableTLSOnSocket(Socket socket) {
             ((SSLSocket) socket).setEnabledProtocols(new String[] {
                 "TLSv1.2"
             });
-            if (portNumber == 443 && Conscrypt.isConscrypt((SSLSocket) socket)) {
```

Comment:
Why was this port number check removed?

Reply:
Because port 443 nows corresponds to socket having Conscrypt as security provider and vice-versa rendering the check redundant